src/parser/core/modules/Tincture.ts

```diff
@@ -0,0 +1,12 @@
+import {t} from '@lingui/macro'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+
+export class Tincture extends BuffWindowModule {
+	static handle = 'tincture'
+	static title = t('core.tincture.title')`Tinctures`
```

Comment:
This could maybe be Medicine or Medicated. Newer players might get confused with pots vs infusions vs tinctures, although we do only care about level cap so tinctures is prob fine. JP calls them all "medicine" anyway, I imagine DE is prob similar.

Reply:
I'm sure a lot of people call them potions (even though actual Potions don't give you Medicated), but Tinctures is actually what they are at this point.  I'm not tied to the name myself if someone feels strongly.